front-end/src/MeetTheTeam.js

```diff
@@ -2,56 +2,59 @@ import React from 'react'
 import Footer from './Footer'
 import Header from './Header'
 import './MeetTheTeam.css'
-import image from './Images/stock-user.png'
+//import image from './Images/stock-user.png'
```

Comment:
You can probably delete this commented out line of code. Same with most commented out lines of code.